assets/wire/components/fields/DurationLabel.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {gettext} from 'utils';
+import {IArticle} from 'interfaces';
+
+interface IProps {
+    item: IArticle,
+    filterGroupLabels: {
+        [field: string]: string;
+    },
+}
+
+export function DurationLabel (props: IProps) {
+    const {item, filterGroupLabels} = props;
+
+    if (item.extra?.duration == null) {
+        return null;
+    }
+
+    function zeroPad(number: number) {
```

Comment:
imo would be better to make it return string :
`zerePad(number: number) : string {`